### PR TITLE
feat(core): support using native scrollbar

### LINKED ISSUE #1525: 🚀 - ability to disable or alter custom scollbar
### Which @taiga-ui/* package(s) are relevant/releated to the feature request?

_No response_

### Description

I think the title is pretty self-explanatory

### PR DESCRIPTION
**Pull request by splincode**
Fixes #1525

image

### CODE REVIEW

**Comment by splincode** [comment]
@waterplea Hello, could you review me?

File: projects/demo/src/modules/components/pull-to-refresh/examples/3/index.html [hunk 1]
```diff
@@ -4,7 +4,8 @@
             appendOnly
             itemSize="50"
             tuiScrollable
-            class="example-viewport tui-zero-scrollbar"
+            class="example-viewport"
+            [class.tui-zero-scrollbar]="!nativeScrollbar"
```
Comment (waterplea) [inline projects/demo/src/modules/components/pull-to-refresh/examples/3/index.html:8]: Please remove this from all demo pages — it is irrelevant to the demo.

File: projects/core/components/scrollbar/scrollbar.component.ts [hunk 2]
```diff
@@ -51,10 +51,10 @@ export class TuiScrollbar {
     protected readonly browserScrollRef = new ElementRef(this.el);
 
     @Input()
-    public hidden = false;
+    public hidden = this.options.mode === 'hidden';
```
Comment (waterplea) [inline projects/core/components/scrollbar/scrollbar.component.ts:54]: Should we deprecate it and instead make a `mode` input?

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/pull-to-refresh/examples/3/index.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/pull-to-refresh/examples/3/index.html]: fixed

File: projects/core/components/scrollbar/scrollbar.component.ts (hunk 2 above)
Comment (splincode) [reply projects/core/components/scrollbar/scrollbar.component.ts:57]: fixed

**Pull request merged**